database/src/main/java/database/service/SqlDatabaseService.java

```diff
@@ -154,6 +154,58 @@ public <T extends BaseEntity> T addNewRecordToTable(T entity) {
         return entity;
     }
 
+    @Override
+    public <T extends BaseEntity> Iterable<T> addNewRecordsToTable(Class<? extends BaseEntity> entityClass,
```

Comment:
In Json db service you check if entities list is null or empty, why don't you do the same here?